nvflare/apis/fl_component.py

```diff
@@ -228,3 +230,198 @@ def _fire_log_event(self, event_type: str, log_tag: str, log_msg: str, fl_ctx: F
         dxo = event_data.to_dxo()
         fl_ctx.set_prop(key=FLContextKey.EVENT_DATA, value=dxo.to_shareable(), private=True, sticky=False)
         self.fire_event(event_type=event_type, fl_ctx=fl_ctx)
+
+
+class FLComponentHelper(FLComponent):
```

Comment:
Please move this class out of fl_component. FLComponent is a low-level basic construct and we want to keep it as simple/small as possible. This Helper thing is one level above the basic component.

This Helper belongs to app_common and not FLARE core.

Reply:
Moved to app common.